workflow/harvesting.md

```diff
@@ -1,17 +1,17 @@
-# Harvesting
+# Data Archiving (aka "Harvesting")
```

Comment:
Do you think we should stick with using "data archiving" to mean the track broadly (research & harvest) or should we align it specifically with the harvesting part?